### DISCUSSION TITLE
Reschedule! One hour later?

### DISCUSSION THREAD

**Discussion by 7yl4r** [USF-IMARS/python-tech-workgroup#35] [Announcements]
I have a recurring meeting 2-3 for the rest of this month that will prevent me from leading this group as scheduled. The easiest solution is to move the python-workgroup time back one hour. 

The new time will be: 3:00-4:30pm ET every Friday.

* 👍 if this time works for you
* 👎 if this time does not work
* comment to suggest a different time
    * 👍 or 👎 on the other commented times too

**Comment by sebastiandig** [comment]
Hi @7yl4r, I think this time will work for me, but I may miss some because of other activities that I attend to. If it moves to sometime Thursday, I am always free.  Another thought, it may be more beneficial for you and I to meet asynchronously because of some of the complexity with the various projects. Let me know your thoughts.
